backend/firebase/firebase.config.js

```diff
@@ -7,19 +7,13 @@ import { getAuth, signInWithEmailAndPassword, createUserWithEmailAndPassword, si
 
 // Your web app's Firebase configuration
 // For Firebase JS SDK v7.20.0 and later, measurementId is optional
-// const express = require('express')
-// const cors = require('cors')
-// require('dotenv').config()
-import 'dotenv/config'
 
 
 
-// console.log('Testing the process.env:',process.env)
-const testKey = import.meta.env.API_KEY
-console.log('Firebase api key:', testKey)
+// console.log('Test:', import.meta.env.REACT_APP_API_KEY)
 
 const firebaseConfig = {
-    apiKey: import.meta.env.API_KEY,
+    apiKey: "AIzaSyCjFjyMbmiEZDeu9Pb6IluxC3cXydh4E2Q",
```

Comment:
Assuming this will be removed later.  This will be forever in the git history, so in a real world scenario you'd need to replace this key.